build/deps.mk

```diff
@@ -51,9 +52,27 @@ THRIFT = $(BIN)/thrift
 PROTOC_LIB = $(LIB)/protoc-$(PROTOC_VERSION)
 PROTOC_ZIP = $(PROTOC_LIB)/protoc.zip
 PROTOC = $(BIN)/protoc
-GEN_BINS = $(THRIFT) $(PROTOC)
+GEN_BINS = $(THRIFT) $(PROTOC) $(RAGEL)
 EXTRA_BINS = $(GLIDE)
 
+RAGEL_TAR = $(LIB)/ragel-$(RAGEL_VERSION)/ragel-$(RAGEL_VERSION).tar.gz
+RAGEL = $(BIN)/ragel
+
+$(RAGEL_TAR):
+	@mkdir -p $(dir $(RAGEL_TAR))
+	curl -L "https://www.colm.net/files/ragel/ragel-$(RAGEL_VERSION).tar.gz" > $(RAGEL_TAR)
+
+$(RAGEL): $(RAGEL_TAR)
+	@mkdir -p $(BIN)
+	@cd $(dir $(RAGEL_TAR)) && \
```

Comment:
Why the echoes? Why not just be consistent with other GEN_BINS? This makes it harder to genericize this in the future. Also is there not a static binary/can you not make one? This will probably significantly increase a completely fresh docker build (hopefully don't have to do this much but)